src/general_manager/bucket/databaseBucket.py

```
@@ -222,8 +294,50 @@ def sort(
+        for k in key:
+            if k in properties:
+                prop = properties[k]
+                if prop.query_annotation is not None:
+                    if callable(prop.query_annotation):
+                        qs = prop.query_annotation(qs)
+                    else:
+                        annotations[k] = prop.query_annotation
+                else:
+                    python_keys.append(k)
+        if not isinstance(qs, models.QuerySet):
+            raise TypeError("Query annotation must return a Django QuerySet")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)